core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -977,6 +977,9 @@ class GroupMetadataManager(brokerId: Int,
     shuttingDown.set(true)
     if (scheduler.isStarted)
       scheduler.shutdown()
+    metrics.removeSensor(GroupMetadataManager.LoadTimeSensor)
+    metrics.removeSensor("OffsetCommits")
```

Comment:
nit: maybe we could add a constant for these two similar to `GroupMetadataManager.LoadTimeSensor`?

Reply:
good idea. will do